skycultures/modern/index.json

```diff
@@ -1675,6 +1675,53 @@
+			 { "symbol": "LS04", "native": "रोहिणी",            "pronounce": "Rohini",     "english": "Red One"},
+			 { "symbol": "LS05", "native": "मृगशिर",            "pronounce": "Mrigashira", "english": "Deer's Head"},
+			 { "symbol": "LS06", "native": "आर्द्रा",             "pronounce": "Ardra",      "english": "Storm God"},
+			 { "symbol": "LS07", "native": "पुनर्वसु ",            "pronounce": "Punarvasu",  "english": "Two Restorers of Goods"},
+			 { "symbol": "LS08", "native": "पुष्य",              "pronounce": "Pushya",    "english": "Nourisher"},
+			 { "symbol": "LS09", "native": "आश्ळेषा/आश्लेषा",       "pronounce": "Āshleshā",   "english": "Embrace"},
+			 { "symbol": "LS10", "native": "मघा",              "pronounce": "Maghā",     "english": "Bountiful"},
+			 { "symbol": "LS11", "native": "पूर्व फाल्गुनी",          "pronounce": "Pūrva Phalgunī", "english": "First Reddish One"},
+			 { "symbol": "LS12", "native": "उत्तर फाल्गुनी",         "pronounce": "Uttara Phalgunī", "english": "Second Reddish One"},
+			 { "symbol": "LS13", "native": "हस्त",              "pronounce": "Hasta",     "english": "Hand"},
+			 { "symbol": "LS14", "native": "चित्रा",              "pronounce": "Chitra",    "english": "Bright One"},
+			 { "symbol": "LS15", "native": "स्वाति",             "pronounce": "Svati",     "english": "Very good"},
+			 { "symbol": "LS16", "native": "विशाखा",            "pronounce": "Vishakha",   "english": "Forked, having branches"},
+			 { "symbol": "LS17", "native": "अनुराधा",           "pronounce": "Anuradha",    "english": "Following rādhā"},
+			 { "symbol": "LS18", "native": "ज्येष्ठा",             "pronounce": "Jyeshtha",   "english": "Eldest, most excellent"},
+			 { "symbol": "LS19", "native": "मूल",              "pronounce": "Mula",       "english": "Root"},
+			 { "symbol": "LS20", "native": "पूर्व आषाढा",          "pronounce": "Purva Ashadha",  "english": "First Invincible"},
+			 { "symbol": "LS21", "native": "उत्तर आषाढा",         "pronounce": "Uttara Ashadha", "english": "Later Invincible"},
+			 { "symbol": "LS22", "native": "श्रवण",             "pronounce": "Shravana",       "english": ""},
+			 { "symbol": "LS23", "native": "श्रविष्ठा/धनिष्ठा",         "pronounce": "Dhanishta/Shravishthā", "english": "Most Famous/Swiftest"},
+			 { "symbol": "LS24", "native": "शतभिष/शततारका",      "pronounce": "Shatabhisha Nakshatra", "english": "Requiring a hundred Physicians"},
+			 { "symbol": "LS25", "native": "पूर्व भाद्रपदा/पूर्व प्रोष्ठपदा",   "pronounce": "Purva Bhadrapada",  "english": "First of the Blessed Feet"},
+			 { "symbol": "LS26", "native": "उत्तर भाद्रपदा/उत्तर प्रोष्ठपदा", "pronounce": "Uttara Bhādrapadā", "english": "Second of the Blessed Feet"},
+			 { "symbol": "LS27", "native": "रेवती",            "pronounce": "Revati",            "english": "Prosperous"}, 
+			 { "symbol": "LS27", "native": "अभिजित",          "pronounce": "Abhijit",           "english": "Invincible"}]},			 
   "edges_type": "iau",
```

Comment:
really? 

In my view, as the nakshatras represent an own, ~4000 years old culture, they should not be mixed with modern boundaries

Reply:
As said, this is a branch in draft. The Nakshatras will not stay in the modern SC, this is just to have the display immediately after launch. Obvious candidate for integration is Vedic, but you can create your isolated version as well. But let me get the implementation right before moving and fixing the json.